models/topics-models

```diff
@@ -1,4 +1,5 @@
 const db = require("../db/connection.js");
+const apiList = require("../endpoints.json");
```

Comment:
Because you're not making any requests to database etc this could be required straight into the controller, this way is fine though